Solobarine.github.io/portfolio.js

```diff
@@ -0,0 +1,202 @@
+const open = document.querySelector('#bars');
+const imgButton = document.querySelector('.img-button');
+const page = document.querySelector('.pop-up');
+const portfolio = document.querySelector('.the-nav1');
+const about = document.querySelector('.the-nav2');
+const contact = document.querySelector('.the-nav3');
+
+open.addEventListener('click', () => {
+  if (page.className === 'pop') {
+    page.className = '';
+  } else {
+    page.className = 'pop';
+  }
+});
+
+imgButton.addEventListener('click', () => {
+  if (page.className === '') {
+    page.className = 'pop';
+  } else {
+    page.className = '';
+  }
+});
+
+portfolio.addEventListener('click', () => {
+  if (page.className === '') {
+    page.className = 'pop';
+  } else {
+    page.className = '';
+  }
+});
+
+about.addEventListener('click', () => {
+  if (page.className === '') {
+    page.className = 'pop';
+  } else {
+    page.className = '';
+  }
+});
+
+contact.addEventListener('click', () => {
+  if (page.className === '') {
+    page.className = 'pop';
+  } else {
+    page.className = '';
+  }
+});
+
+// Popup Window
+const projectItems = {
+  title: 'Keeping track of hundreds of components website.',
+  skillTag: ['html', 'Bootstrap', 'Ruby on Rails'],
+  summary: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.',
+  liveButton: 'See Live',
+  sourceButton: 'See Source',
+};
+const [var1, var2, var3] = projectItems.skillTag;
+// Create Dynamic Page
+const grand = document.querySelector('.show-inactive');
+const parent = document.querySelector('.featured-item');
+const exit = document.createElement('button');
+const heading = document.createElement('h2');
+const li = document.createElement('ul');
+const html = document.createElement('li');
+const bootstrap = document.createElement('li');
+const rails = document.createElement('li');
+const project = document.createElement('div');
+const text = document.createElement('p');
+const live = document.createElement('button');
+const liveImg = document.createElement('span');
+const source = document.createElement('button');
+const sourceImg = document.createElement('span');
+
+const btn1 = document.querySelector('.btn-1');
+const btn2 = document.querySelector('.btn-2');
+const btn3 = document.querySelector('.btn-3');
+const btn4 = document.querySelector('.btn-4');
+const btn5 = document.querySelector('.btn-5');
+const btn6 = document.querySelector('.btn-6');
```

Comment:
- [ ] Your popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, text, margins).

Your design 
![image](https://user-images.githubusercontent.com/43877689/180437940-adbe1e1d-bd9d-43cb-8e24-1e4668302fb0.png)

Figma design 
![image](https://user-images.githubusercontent.com/43877689/180438210-500906b4-c50a-48ab-a5fc-b2f3236d06da.png)